domain/src/test/java/org/oppia/android/domain/oppialogger/analytics/BUILD.bazel

```diff
@@ -289,4 +289,35 @@ oppia_android_test(
     ],
 )
 
+oppia_android_test(
+    name = "FeatureFlagsLoggerTest",
+    srcs = ["FeatureFlagsLoggerTest.kt"],
+    custom_package = "org.oppia.android.domain.oppialogger.analytics",
+    test_class = "org.oppia.android.domain.oppialogger.analytics.FeatureFlagsLoggerTest",
+    test_manifest = "//domain:test_manifest",
+    deps = [
+        ":dagger",
+        "//domain/src/main/java/org/oppia/android/domain/oppialogger/analytics:feature_flags_logger",
+        "//domain/src/main/java/org/oppia/android/domain/oppialogger/analytics:prod_module",
+        "//domain/src/main/java/org/oppia/android/domain/oppialogger/survey:survey_events_logger",
```

Comment:
I haven't seen a dependency on the survey logger (or noticed a relevance in this PR). Am I missing something?